### DISCUSSION TITLE
[Nhận task]

### DISCUSSION THREAD

**Discussion by trungdoublelift** [Runway-Club/Runway-CV-Builder#62] [Task Delivery]
Tôi muốn nhận task Lưu dữ liệu người dùng nhập trong input create CV bằng firebase

**Comment by nguyentantrung2000** [comment]
Thông tin task:
Lưu các thông tin người dùng nhập 
Yêu cầu:
Luư trữ được trên firbase 
Deadline 11:59PM 12/8/2021

**Reply by trungdoublelift** [reply]
OK
